migration/migrator/migrations/course/20240626171518_remove_poll_status.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE ONLY polls DROP COLUMN IF EXISTS status")
```

Comment:
One more quick question: is `ONLY` needed here for some reason?  (the same comment applies to the down migration as well)